app/controllers/SubscriptionController.scala

```diff
@@ -55,7 +54,7 @@ class SubscriptionController @Inject() (
         valid = subscriptionRequest =>
           for {
             response <- subscriptionConnector.sendSubscriptionInformation(subscriptionRequest)
-            _        <- sendAuditEvent(request.affinityGroup, subscriptionRequest, response)
+            _        <- sendAuditEvent(subscriptionRequest, response)
```

Comment:
Is sendSubscriptionInformation Returning audit obj? if not then we have to wrap it in audit model